.env.example

```
@@ -74,4 +74,15 @@ TG_MESSAGE_MAX_LENGTH="3000" # макс. длина текста
 TG_AUDIO_MAX_BYTES="8388608"  # 8 МБ
 TG_AUDIO_MAX_DURATION_SEC="120" # 2 минуты
 # Таймаут скачивания файлов Telegram (мс)
-TG_FETCH_TIMEOUT_MS="25000"
\ No newline at end of file
+TG_FETCH_TIMEOUT_MS="25000"
+
+# S3 / MinIO storage (for voice/audio storage)
+# If you use AWS S3, set AWS_REGION/ACCESS_KEY/SECRET and bucket name.
+# If you use local MinIO from docker-compose, set endpoint to http://localhost:9000 and force path style to true.
+AWS_S3_BUCKET="synoro-bucket"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)